<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisconnect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect!</string>
    </property>
   </widget>
   <widget class="QSlider" name="sldThrotle">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QDial" name="dialPolling">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblThortle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Throtle</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRudder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rudder</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAileron">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aileron</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblElevator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator</string>
    </property>
   </widget>
   <widget class="QLabel" name="Elevation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevation</string>
    </property>
   </widget>
   <widget class="QSlider" name="sldRudder">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sldAileron">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sldElevator">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblPollingTime">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Polling Time: 10</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numElevation">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>557</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
